src/engine/idTable/IdTable.h

```diff
@@ -736,22 +736,35 @@ class IdTable {
   // The input must be some kind of `IdTable`.
   // TODO<joka921> Can/should we constraint this functions by a concept?
   template <typename Table>
-  void insertAtEnd(const Table& table,
-                   std::optional<size_t> beginIdx = std::nullopt,
-                   std::optional<size_t> endIdx = std::nullopt) {
-    AD_CORRECTNESS_CHECK(table.numColumns() == numColumns());
+  void insertAtEnd(
```

Comment:
The documentation must also talk about the newly added arguments (what do they do, what are the requirements